### PR TITLE
Fixes #10066: Scary message about OpenSSL on SLES when running rudder agent update

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/10066

### CODE REVIEW

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -189,7 +189,7 @@ Requires: dmidecode
 ### SLES 11 OSes come with OpenSSL 0.9.8h which is recent enough.
 ##
 # Reference for suse_version : https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto
-%if 0%{?suse_version} && 0%{?suse_version} < 1100
+%if 0%{?suse_version} && 0%{?suse_version} < 1140
```
Comment (amousset) [inline rudder-agent/SPECS/rudder-agent.spec]: I don't know if there will be a SP5, but it seems to me the actual condition is >= 1200

Comment (peckpeck) [inline rudder-agent/SPECS/rudder-agent.spec]: it seems logical, it's just that we didn't do that for the other tests

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/SPECS/rudder-agent.spec [hunk 2]
```diff
@@ -144,12 +144,12 @@ Requires: dmidecode
 
 ## 3 - SLES: No LMDB yet
 # Reference for suse_version : https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto
-%if 0%{?suse_version} && 0%{?suse_version} < 1140
```
Comment (amousset) [inline rudder-agent/SPECS/rudder-agent.spec]: This one is a change in OpenSuSE 11.4

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/SPECS/rudder-agent.spec [hunk 3]
```diff
@@ -149,7 +149,7 @@ Requires: pmtools
 %define use_system_lmdb false
 %endif
 
-%if 0%{?suse_version} && 0%{?suse_version} >= 1140
```
Comment (amousset) [inline rudder-agent/SPECS/rudder-agent.spec]: This was was also OpenSuSE 11.4 specific (it is the other part of the first one)

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**